tests/Application/Commands/ReconcilePastPayoutsTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MatchBot\Tests\Application\Commands;
+
+use MatchBot\Application\Commands\ReconcilePastPayouts;
+use MatchBot\Application\Messenger\StripePayout;
+use MatchBot\Tests\Application\DonationTestDataTrait;
+use MatchBot\Tests\TestCase;
+use Prophecy\Prophecy\ObjectProphecy;
+use Psr\Log\NullLogger;
+use Stripe\Event;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\Lock\LockFactory;
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Messenger\RoutableMessageBus;
+use Symfony\Component\Messenger\Stamp\BusNameStamp;
+use Symfony\Component\Messenger\Stamp\TransportMessageIdStamp;
+
+class ReconcilePastPayoutsTest extends TestCase
+{
+    use DonationTestDataTrait;
+
+    public function testRun(): void
+    {
```

Comment:
this test is fine and we might as well keep it for now and delete with ReconcilePastPayouts when that's been run. But really ReconcilePastPayouts seems simple enough to not need testing, especially if we did a search and replace in the source code to change the CSV string into a hard-coded PHP array.